src/broker/cas_execute.c

```diff
@@ -3228,6 +3237,7 @@ ux_cursor_close (T_SRV_HANDLE * srv_handle)
     }
 #if defined(CAS_FOR_CGW)
   cgw_cursor_close (srv_handle->cgw_handle->hstmt);
```

Comment:
no need to check error?

Reply:
Thanks for your comment.
Added error check code.
if (cgw_cursor_close (srv_handle->cgw_handle->hstmt) > -1)
    {
      srv_handle->cgw_handle->hstmt = NULL;
    }